lib/readable.rb

```diff
@@ -1,58 +1,31 @@
 module Readable
   def write(file_path, type)
     if type == 'encrypt'
-      File.write(file_path, "#{encryption}, #{key}, #{date}\n", mode: 'a')
+      File.write("./lib/#{file_path}", "#{encryption}\n", mode: 'a')
     elsif type == 'decrypt'
-      File.write(file_path, "#{message}, #{key}, #{date}", mode: 'a')
+      File.write("./lib/#{file_path}", "#{message}\n", mode: 'a')
     end
   end
 
   def status_message(file_path)
-    "Created '#{File.basename(file_path)}' with the key #{key} and the date #{date}"
+    "Created '#{File.basename("./lib/#{file_path}")}' with the key #{key} and the date #{date}"
   end
 
-  def shift_type(type, info)
-    info = info.flatten
+  def shift_type(type, message, *data)
+    message = message.flatten
```

Comment:
Also no need to flatten this since it is no longer an array rcvd from the working line in text file